client/src/locales/en-US.json

```diff
@@ -1227,7 +1227,9 @@
         "loggedInOrgs": "Connected organizations",
         "myOrgs": "My organizations",
         "invite": "Invite a member",
-        "manage": "Manage"
+        "manage": "Manage",
+        "goToLogin": "Switch organization",
```

Comment:
This is a bit misleading since we can switch to another logged in organization from this popover. But I'm not sure what to write instead, it would be nice to mention that it's to log in to a logged-off organization.